src/inverseRobot.js

```diff
@@ -7,7 +7,25 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+  const propertyValues = Object.values(robot);
```

Comment:
Consider using a Set to track duplicates. A Set automatically ensures all its elements are unique, so you can use it to check for duplicates in a single pass, which is more efficient than the nested loop approach.